src/DynamoRevit/DynamoRevit.cs

```diff
@@ -593,8 +593,6 @@ private static void InitializeCore(DynamoRevitCommandData commandData)
         private static void InitializeAssemblies()
         {
             RevitAssemblyLoader.LoadAll();
-            AppDomain.CurrentDomain.AssemblyResolve +=
```

Comment:
Is the RaaS the only analysis assembly that would cause this callback to be called?

Reply:
@ColinDayOrg  Good question. I believe so, the call of `Analyze.Render.AssemblyHelper.ResolveAssemblies` is actually in `DynamoRaaSHelper` which is the dll we are removing from the project by touching `DynamoRevit.csproj`. As the reference is removed, the actual code should not be called again.